README.md

```diff
@@ -39,82 +41,111 @@ $ npm run example:application # start embedded application
 
-- [**React**](http://reactjs.org/) - A JavaScript library for building user interfaces.
-- [**Webpack**](https://webpack.js.org/) - Component bundler.
-- [**React testing library**](https://testing-library.com/) - Simple and complete testing utilities that encourage good testing practices.
-- [**Emotion**](https://emotion.sh/) - Library designed for writing css styles with JavaScript.
-- [**Babel**](https://babeljs.io/) - Write next generation JavaScript today.
-- [**Jest**](https://facebook.github.io/jest) - JavaScript testing framework used by Facebook.
-- [**Testcafe**](https://devexpress.github.io/testcafe/) - A node.js tool to automate end-to-end web testing.
-- [**Percy**](http://percy.io/) - Visualisation testing tool.
-- [**ESLint**](http://eslint.org/) - Make sure you are writing a quality code.
-- [**Prettier**](https://prettier.io/) - Enforces a consistent style by parsing your code and re-printing it.
-- [**Typescript**](https://www.typescriptlang.org/) - JavaScript superset, providing optional static typing
-- [**Circle CI**](https://circleci.com/) - Automate tests and linting for every push or pull request.
-- [**Storybook**](https://storybook.js.org/) - Tool for developing UI components in isolation with documentation.
-- [**Debug**](https://github.com/visionmedia/debug) - JS debugging utility that works both in node.js and browsers.
 
-## Changing stuff
+## Uninstallation
 
-### Removing Emotion
+### Removing emotion
 
-- Uninstall npm packages
+To uninstall `npm` packages with emotions, use the command below:
 
 ```sh
 $ npm uninstall @emotion/core @emotion/styled @emotion/babel-preset-css-prop
 ```
 
-- Remove `@emotion/babel-preset-css-prop` from Babel presets (configuration is made in package.json file)
-- Remove `@emotion/core` from Typescript configuration (in tsconfig.json file)
-- Remove `.storybook/webpack.config.js` (file created only for emotion css property support in storybook)
+- The uninstallation process will remove `@emotion/babel-preset-css-prop` from Babel presets (configuration is made in the `package.json` file).
+- It will remove `@emotion/core` from TypeScript configuration (in the `tsconfig.json` file).
+- It will also remove `.storybook/webpack.config.js` (a file created only for emotion CSS property support in storybook).
 
-### Removing Examples
+### Removing examples
 
-- Uninstall npm packages
+To uninstall `npm` packages with examples, use the command below:
 
 ```sh
 $ npm uninstall qrcode.react @types/qrcode.react
 ```
 
-- Remove `examples` folder completely
-- Remove line 5 with `../example` context from `.storybook/config.js` (so that storybook won't look for example files)
+- The uninstallation process will remove the `examples` folder completely.
+- It will remove line 5 with the `../example` context from `.storybook/config.js`, so that the storybook won't look for example files.
 
 ## License
 
-GPL-3.0
+The React template for the decentralized renderer is under the GPL-3.0 license.
```

Comment:
Yes can we update this to `Apache 2.0 license`?

Reply:
Yes